cli/utilities/utils.py

```diff
@@ -28,11 +28,20 @@ def get_running_containers(node, expr=None, **kw):
     return node.exec_command(cmd=cmd, **kw)
 
 
+def os_major_version(node, **kw):
+    """Get OS major release version
+    Args:
+        node (ceph): Ceph node object
+    """
+    cmd = r"cat /etc/os-release | tr -dc '0-9.'| cut -d \. -f1"
```

Comment:
I always found this useful `eval $(cat /etc/os-release) ; echo $VERSION_ID`

Reply:
@psathyan `VERSION_ID` returns `major.minor` version. If we need to use above command then it should be `eval $(cat /etc/os-release) ; echo $VERSION_ID | cut -d \. -f1`